web/js/modules/date/util.js

```diff
@@ -24,13 +30,90 @@ export function tryCatchDate(str, initialState) {
   }
 }
 
+/**
+ * Parse permalink date string and handle max dates if out of valid range or in future
+ *
+ * @method parsePermalinkDate
+ * @param  {String} date string (now or nowMinusSevenDays if B side for compare mode)
+ * @param  {String} date string in querystring
+ * @param  {String} layerParameters (A or B depending if compare mode is active)
+ * @param  {Object} config object
+ * @returns {Object} date object
+ */
+export function parsePermalinkDate(defaultStr, str, layerParameters, config) {
+  let time = tryCatchDate(str, defaultStr);
+  if (time instanceof Date) {
+    const startDate = new Date(config.startDate);
+    if (time < startDate) {
+      time = startDate;
+    } else if (time > defaultStr) {
```

Comment:
Looks like `time > defaultStr` is comparing a `Date` to a date `String`. Not sure if that works? Might be worth investigating.

Reply:
Good catch, thanks! There was a wrong method description and variable name that I addressed.